tests/conftest.py

```diff
@@ -98,12 +98,20 @@ class Student(Base):
             secondary=student_course,
             backref=backref("students", lazy="dynamic"),
         )
-        # Test complex column property
-        course_count = column_property(
-            sa.select([sa.func.count(student_course.c.course_id)]).where(
-                student_course.c.student_id == id
+        try:
+            subquery = (
```

Comment:
Looks good - can we avoid repeating:
```
sa.select([sa.func.count(student_course.c.course_id)])
.where(student_course.c.student_id == id)
```
and just write that once on the top ?

Reply:
Hi @AbdealiJK ,

Made the suggested changes